i18n/en.json

```diff
@@ -4706,6 +4706,22 @@
     "id": "model.websocket_client.connect_fail.app_error",
     "translation": "Unable to connect to the WebSocket server."
   },
+  {
```

Comment:
Propose minor adjustments:


```
  {
    "id": "model.service_terms.is_valid.id.app_error",
    "translation": "Invalid term of service id.”
  },
  {
    "id": "model.service_terms.is_valid.create_at.app_error",
    "translation": "Missing required term of service property: create_at.”
  },
  {
    "id": "model.service_terms.is_valid.user_id.app_error",
    "translation": "Missing required terms of service property: user_id."
  },
  {
    "id": "model.service_terms.is_valid.text.app_error",
    "translation": "Invalid terms of service text."
  },
```

Reply:
Done. Also removed DeleteAt and EditAt fields from ServiceTerms model.